README.md

```
@@ -104,6 +105,106 @@ docker compose up -d
+      properties:
+        fileId:
+          description: ""
+          type: string
+      required:
+        - fileId
+      type: object
+```
+
+Recommended workflow:
+
+1. Add `tools.file` to the server's config (see [`mcpServers.<name>.tools`](#mcpservernametools)) and run `manifold openapi generate -c config`.
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)